production/jaeger-production-template.yml

```diff
@@ -0,0 +1,345 @@
+#
+# Copyright 2017 The Jaeger Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+#
+
+parameters:
+- description: The name of the Jaeger Service.
+  displayName: Jaeger Service Name
+  name: JAEGER_SERVICE_NAME
+  required: true
+  value: jaeger
+- description: The namespace where to install the Jaeger instance
+  displayName: Namespace
+  name: NAMESPACE
+  required: true
+  value: jaeger-infra
+- description: The Cassandra Keyspace for the Jaeger data
+  displayName: Jaeger Cassandra Keyspace
+  name: KEYSPACE
+  required: true
+  value: jaeger_v1_dc1
+- description: The Jaeger image version to use
+  displayName: Image version
+  name: IMAGE_VERSION
+  required: false
+  value: 9f3984cd
+
+apiVersion: v1
+kind: Template
+labels:
+  app: jaeger
+  template: jaeger-template
+  jaeger-infra: template
+message: 'A Jaeger service has been created in your project.'
+metadata:
+  name: jaeger-template
+  annotations:
+    description: Jaeger Distributed Tracing Server
+    iconClass: icon-go-gopher
+    openshift.io/display-name: Jaeger
+    tags: instant-app,tracing,opentracing,jaeger
+  labels:
+    app: jaeger
+    name: jaeger-infra
+objects:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    namespace: "${NAMESPACE}"
```

Comment:
I'm maybe too much pedantic, but quotes are not necessary here also in apiVersions and other definitions